### DISCUSSION TITLE
how to set global config with neovim lspconfig pyright-langserver

### DISCUSSION THREAD

**Discussion by cxwx** [microsoft/pyright#7903] [General]
simular question https://github.com/microsoft/pyright/issues/176

I could use `pyright -p config.toml $filename`  to set config 
but the neovim lspconfig
cmd = `pyright-langserver --stdio`

how to set a configure file as `~/.pyproject.toml` for the langserver pyright-langserver?

**Comment by erictraut** [comment]
Converting to a discussion since this is a question.

**Reply by erictraut** [reply]
When using the language server, you can specify settings via the language server interface. You can specify global, per-project, or per-user [settings](https://microsoft.github.io/pyright/#/settings).

If you want to use a config file, you should place it in the root directory of your project. It's unlikely that we'd add support for specifying a different location for the config file.

**Reply by cxwx** [reply]
I want a global configure, but the https://github.com/microsoft/pyright/issues/176said it's not supported
I've tried to create ~/.pyrightconfig.json, but it doesn't work

**Reply by erictraut** [reply]
That's correct. This isn't supported.

**Reply by erictraut** [reply]
A symbolic link might work.
